spec/definitions/V0CommandArgs.yaml

```diff
@@ -0,0 +1,42 @@
+title: Command args
+type: object
+properties:
+  wipe_external_storage:
+    type: boolean
+    default: false
+  wipe_FRP:
+    type: boolean
+    default: true
+  brightness_value:
+    type: integer
+    minimum: 1
+    maximum: 100
+  stream:
+    $ref: '#/definitions/SettingsVolumeStreamEnum'
+  volume_level:
+    type: integer
+    minimum: 0
+    maximum: 100
+  wifi_state:
+    type: boolean
+  bluetooth_state:
```

Comment:
We do not have most the commands implemented in new framework. Some the fields here are not present in new one. this will wrong documentation. we need only keep the ones required for supported command.